client/packages/system/src/Encounter/ListView/ListView.tsx

```diff
@@ -18,22 +19,24 @@ import { EncounterFragment, useEncounter } from '@openmsupply-client/programs';
 
 const EncounterListComponent: FC = () => {
   const {
-    updateSortQuery,
     updatePaginationQuery,
-    queryParams: { sortBy, page, first, offset },
+    queryParams: { page, first, offset },
```

Comment:
can you revert these changes please. We want to use the URL query for the sort params!
I've found the bug in the URL query params - there's actually a wider problem.. where all table sort change of direction is failing!

Reply:
Right. Good to fix the original cause at first place. 3a6e872be